includes/Render.php

```diff
@@ -35,14 +35,20 @@ protected function hooks() {
             add_filter( 'woocommerce_get_price_html', [ $this, 'button_html' ], 11, 2 );
             add_action( 'woocommerce_single_variation', [ $this, 'hide_single_variation_add_to_cart' ] );
         }
-
+        
         if ( wcp_is_on( Constants::OUT_OF_STOCK ) ) {
             add_filter( 'woocommerce_get_price_html', [ $this, 'out_of_stock' ], 10, 2 );
         }
 
         if ( wcp_is_on( Constants::MINIMUM_STOCK_THRESHOLD ) ) {
             add_filter( 'woocommerce_get_price_html', [ $this, 'woocommerce_low_on_stock' ], 10, 2 );
         }
+
+        if ( get_option( Constants::WCP_ENABLED_TAXONOMY ) ) {
```

Comment:
Use `wcp_is_on` instead of get_options. `wcp_is_on` already check if the value is true or false and returns a boolean value.

Reply:
Using wcp_is_on does not work